docs/core/tutorials/top-level-templates.md

```diff
@@ -77,3 +79,11 @@ You can also add a [`<Using>`](../project-sdk/msbuild-props.md#using) item in yo
   <Using Remove="System.Net.Http" />
 </ItemGroup>
 ```
+
+## Use the old program style
```

Comment:
I agree with @tdykstra 's general comments to include in this section. Then, this is ready to :shipit: